Propose changes to the definition of browsers (#177)
orcwg/cra-hub: product-definitions/input-to-draft-regulation.md

          Embedded browsers are browsers that are intended for integration into another system or application.
        </p>
      </td>

>       <td>Command-line tools (such as Curl) and other web clients used for software development and testing also "enable end users to access and interact with web content hosted on servers that are connected to networks such as the Internet" through the <a href="https://en.wikipedia.org/wiki/Command-line_interface">command-line interface</a> or through other means. Yet they are not browsers per se. In particular they do not <em>render</em> the web content, but merely fetch it. They are used by technically savvy users to perform specific tasks (like looking at HTTP headers for example) and their interaction capabilities are limited. We suggest amending the description to account for this.</td>

Reviewer: ```suggestion
<td><!-- Comments --><p> This category and definition may be broader than intended. If the focus is on the software application that enables a user to navigate, interact with, and transmit information over the internet, then this is specific functionality of a true browser. Command-line tools (such as Curl) and other web clients used for software development and testing also "enable end users to access and interact with web content hosted on servers that are connected to networks such as the Internet" through the <a href="https://en.wikipedia.org/wiki/Command-line_interface">command-line interface</a> or through other means. Yet they are not browsers per se. In particular they do not <em>render</em> the web content, but merely fetch it. They are used by technically savvy users to perform specific tasks (like looking at HTTP headers for example) and their interaction capabilities are limited. We suggest amending the description to account for this. </p> <p> Embedded browsers may have limited functionality for interacting with web content, and may also be constrained to the existing security controls, mechanisms, and due diligence afford to the system for which they are embedded. Review the intent and functional scoping of the "embedded" browser as a Class I and its requirements to ensure the desired outcome of categorizing embedded browsers is correct as opposed to standalone browsers. </p> </td>
```

>       <td><p>In the context of this category of products, browsers are software products with digital elements that enable end users to access and interact with web content hosted on servers that are connected to networks such as the Internet <ins>through a graphical user interface (GUI) or assistive technology.</ins></p>

Author: What about:

```suggestion
      <td><p>In the context of this category of products, browsers are software products with digital elements that fetch and render web content hosted on servers that are connected to networks such as the Internet, and enable end users to access and interact with it through a graphical user interface (GUI) or assistive technology.</p>
```

>       <p>This category does not include command-line tools and web clients intended to be used for software development and testing.</p>

Reviewer 2: ```suggestion
      <p>This category does not include command-line tools and web clients intended to be used primarily for software development and testing.</p>
```
(I think we should make sure we don't create a loophole for browsers that include excellent development functionality.)

          Embedded browsers are browsers that are intended for integration into another system or application.
        </p>
      </td>

>       <td>Command line tools such as Curl "enable end users to access and interact with web content hosted on servers that are connected to networks such as the Internet" through the <a href="https://en.wikipedia.org/wiki/Command-line_interface">command-line interface</a>. Yet they're not technically browsers. They are used by technically savvy users to perform specific tasks (like looking at HTTP headers for exammple) and their interaction capabilities are limited. We suggest ammending the description to account for this.</td>

Author: ```suggestion
      <td>Command-line tools (such as Curl) and other web clients used for software development and testing also "enable end users to access and interact with web content hosted on servers that are connected to networks such as the Internet" through the <a href="https://en.wikipedia.org/wiki/Command-line_interface">command-line interface</a> or through other means. Yet they are not browsers per se. They are used by technically savvy users to perform specific tasks (like looking at HTTP headers for example) and their interaction capabilities are limited. We suggest amending the description to account for this.</td>
```

>       <td>In the context of this category of products, browsers are software products with digital elements that enable end users to access and interact with web content hosted on servers that are connected to networks such as the Internet <ins>through a graphical user interface (GUI) or assistive technology.</ins></td>

Author: ```suggestion
      <td><p>In the context of this category of products, browsers are software products with digital elements that enable end users to access and interact with web content hosted on servers that are connected to networks such as the Internet <ins>through a graphical user interface (GUI) or assistive technology.</ins></p>
      <p><ins>This category does not include command-line tools and other web clients intended to be used for software development and testing.</ins></p>
      </td>
```
